ldi-core/ldes-client-v2/tree-node-supplier/src/main/java/ldes/client/treenodesupplier/repository/inmemory/InMemoryMemberRepository.java

```diff
@@ -0,0 +1,39 @@
+package ldes.client.treenodesupplier.repository.inmemory;
+
+import ldes.client.treenodesupplier.domain.entities.MemberRecord;
+import ldes.client.treenodesupplier.repository.MemberRepository;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+public class InMemoryMemberRepository implements MemberRepository {
+
+	List<MemberRecord> unprocessed = new ArrayList<>();
```

Comment:
can these be private and final?

Reply:
private, yes
final, no, since we destroy state by re-initializing.